shared/actions/json/route-tree.json

```diff
@@ -29,6 +14,10 @@
       "_description": "ONLY used by the new nav. Navigates up to this route if it already exists, noops otherwise.",
       "routeName": "string"
     },
+    "switchLoggedIn": {
+      "_description": "ONLY used by the new nav. Switch login stacks",
```

Comment:
is it time to delete the "only used by thee new nav" comments here?

Reply:
we'll be doing a larger refactor of this soon